### DISCUSSION TITLE
Queue Mod Install On Next Reboot

### DISCUSSION THREAD

**Discussion by MarkimusDev** [gameserverapp/Platform#488] [Suggestions]
Currently we can not install mods without bringing the server down, but for clusters that have different restart times, this can be an inconvenience.

I propose the ability to "Queue Mod Install", give it a list of ids, and the next time the servers chosen are rebooted, the mod is installed first.

**Comment by mvaessen** [comment]
Duplicate of https://github.com/gameserverapp/Platform/discussions/389
